R/print.R

```diff
@@ -34,21 +37,22 @@ format.errors = function(x,
                          ...)
 {
   stopifnot(notation %in% c("parenthesis", "plus-minus"))
-
   if (is.null(digits))
     digits <- getOption("errors.digits", 1)
   digits <- if (digits == "pdg") digits_pdg(.e(x)) else rep(digits, length(x))
-
   scipen <- getOption("scipen", 0)
   prepend <- rep("", length(x))
   append <- rep("", length(x))
 
   e <- signif(.e(x), digits)
   nulle <- e == 0 & !is.na(e)
-  xexp <- ifelse(.v(x) == 0, get_exponent(e) + 1, get_exponent(x))
-  value_digits <- ifelse(e, digits - get_exponent(e), digits)
+  e_expon = get_exponent(e)
```

Comment:
Please, call it `eexp` and use the `<-` operator to match the style of the rest.